07week/towersOfHanoi/script.js

```diff
@@ -3,20 +3,75 @@
 class TowersOfHanoi extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      grabbedBlock: 0,
+      a: [100,75,50,25],
+      b: [],
+      c: [],
+    }
+    this.game = this.game.bind(this);
+  }
+
+  game(event){
+    let stackLetter = event.target.getAttribute("data-stack");
+    let grabbedItem = this.state.grabbedBlock;
+    let selectedBlock = this.state[stackLetter];
+    let checkA = this.state.a;
+    let checkB = this.state.b;
+    let checkC = this.state.c;
+
+
+    console.log(checkB.length);
+    console.log(grabbedItem);
+    console.log(selectedBlock);
+    console.log(selectedBlock[selectedBlock.length - 1]);
+
+
+//this if-statement controls the logic of the game play. If this.state.grabbedBlock is zero, you can grab a block. If you've grabbed a block, you can drop it if you drop it on a larger block
+    if(grabbedItem == 0){
```

Comment:
this results in some bugs when you select an empty  stack at the beginning of the game.